test/vanilla/ignite-new.test.ts

```diff
@@ -1,20 +1,9 @@
 import { filesystem } from "gluegun"
-import * as tempy from "tempy"
+import * as path from "path"
```

Comment:
I think I'd rather keep the `tempy` solution. We used to have a local gitignored folder for this back in Ignite 1.0 days, but went with the tempy solution instead. I forget the exact reasoning, but it had to do with running tests in parallel and not getting conflicts between integration tests.

When I'm debugging the integration tests, I comment out the `filesystem.remove(tempDir)` in the `afterEach` block and instead do `console.error(tempDir)`. It'll leave the folder as-is and tell me where it is, so I can go examine it.

Reply:
Right now, we have one giant test for the new command, so there isn't any conflicts. If we add more tests, what I've done in past projects is add a unique ID to the `temp` folder for each artifact, either with a UUID or a timestamp

Reply:
I'm not a huge fan of needed to comment and uncomment code if I can help it, there are a lot of times I've accidentally committed debugging code.

Originally I was trying to get this work in Windows, that led me to the `temp` solution. I have a small SSD drive for my OS, and a data drive, so that may be part of the `cd` command is failing. However, I have switched a WSL Ubuntu environment, which may not make `tempy` as much of an issue, where all of WSL is encapsulated on one drive. 

I'll explore this more to see if `tempy` is an issue on WSL

Reply:
`tempy` appears to work fine with WSL. I'll revert these changes then